### ISSUE TITLE
[Windows] Update scroll speed when OS settings change

### ISSUE THREAD

**Issue by loic-sharma** [flutter/flutter#107248]
## Use case

Windows users can change their scroll speed in the OS settings:

![image](https://user-images.githubusercontent.com/737941/177831313-839d163f-c06f-409b-a9c5-2569a20e603b.png)

After https://github.com/flutter/engine/pull/32094, Flutter apps read this scroll speed setting once at startup. However, changing the scroll speed setting doesn't update Flutter apps that are already running.

## Proposal

Make Flutter apps' listen to changes in Windows's scroll speed setting changes. The app's message loop will need to handle `WM_SETTINGCHANGE` messages.

**Comment by wenyue** [comment]
Can we also provide a function to change the multiplier at runtime manually?
I want to provide a settings for user the change the mouse wheel sensitivity at runtime. And I don't want to require the user to change the system configuration.

**Comment by loic-sharma** [comment]
@wenyue This issue is specific to the Windows platform. These issues seem like a better match for what you're asking:

1. https://github.com/flutter/flutter/issues/63389
2. https://github.com/flutter/flutter/issues/85111

**Comment by wenyue** [comment]
No, I hope flutter can modify it globally instead of modifying each widget individually.
